test/jasmine/tests/treemap_test.js

```diff
@@ -1225,7 +1225,7 @@ describe('Test treemap tweening:', function() {
                 'M284.375,188.5L548.375,188.5L548.375,308.5L284.375,308.5Z'
             );
             _assert('move B text to new position', 'transform', 'B', [220.25126, 0]);
-            _assert('enter b text to new position', 'transform', 'b', [287.375195, 5]);
+            _assert('enter b text to new position', 'transform', 'b', [284.66071, 35714285714286]);
```

Comment:
That `35714285714286` y-translate coordinates looks odd. Is this node support to go way off the screen?

Reply:
@etpinard No that should stay on the screen.
I have no idea why we need that only for the test to pass on the CI.

Reply:
> I have no idea why we need that only for the test to pass on the CI.

Can you try to find out why?

Reply:
... by first comparing this `35714285714286` value with the before and after transition values

Reply:
OK. Let me find out.

Reply:
That is related to using one `space character` instead of a `blank string`.
 With that the test is OK.

Reply:
So, what y-translate value do we get if we use a blank string?

Reply:
[287.375195, 0] if we use blank. But then the transition doesn't look good.

Reply:
Ok - well this is hardly a satisfactory answer, but hey the transition looks better now, so I'll stop complaining.